app/mailers/user_mailer.rb

```diff
@@ -124,7 +124,7 @@ def friendship_notification(friendship)
   def note_comment_notification(comment, recipient)
     with_recipient_locale recipient do
       @noteurl = note_url(comment.note)
-      @place = Nominatim.describe_location(comment.note.lat, comment.note.lon, 14, I18n.locale)
+      @place = Nominatim.describe_location(comment.note.lat.to_f, comment.note.lon.to_f, 14, I18n.locale)
```

Comment:
This change defeats the purpose of using `DelegateClass`, because e.g. note.lat is supposed to be indistinguishable from it being a float, apart from how we format it as a string. So I'd really like to avoid having to call `.to_f` on something that's supposed to act like a float. And I also think this might break the formatting too, since to_s will be called on the float, not on the `Coord`

I spent several hours figuring out what is going on, and narrowed it down to https://github.com/rails/rails/pull/49466 and there is a fix already at https://github.com/rails/rails/pull/49576 which was merged earlier today. We should either wait for that fix to be released, or revert this commit when it happens.